hack/activate-env.sh

```diff
@@ -40,7 +41,10 @@ _activateEnv() {
   set -u
 
   echo "--> Calling conda-unpack..."
-  conda-unpack
+  if [ $quiet ]; then
+    conda-unpack --quiet
```

Comment:
To keep things simple, should we always run `conda-pack` in quiet mode and remove the `_quiet` arg?

Reply:
True! Would that be ok for everyone? Does anyone use those logs?

Reply:
@adriangonz ^

Reply:
It should be alright to remove them :+1: